Wrath/Rogue.lua

```diff
@@ -917,7 +917,7 @@ spec:RegisterAbilities( {
 
     -- Enrages you, increasing all damage caused by 5%.  Requires a bleed effect to be active on the target.  Lasts 1 min.
     hunger_for_blood = {
-        id = 51662,
+        id = 63848,
```

Comment:
I saw that for some abilities and auras, the IDs were different between the ability and the aura that ability confers, and then for other abilities, they were the same. I'm not sure if this should be the same ID or different?